specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualWan.json

```diff
@@ -3993,22 +3993,35 @@
     },
     "P2SVpnConnectionHealthRequest": {
       "properties": {
-        "p2sVpnConnectionIdsOrVpnUserNames": {
+        "p2sVpnConnectionHealthFilters": {
           "type": "array",
           "items": {
-            "type": "string"
+            "type": "p2sVpnConnectionHealthFilter"
```

Comment:
If it's an array of resource, it should be `"$ref": "#/definitions/P2sVpnConnectionHealthFilter"`